packages/types/src/sign-client/engine.ts

```diff
@@ -259,6 +264,7 @@ export interface EnginePrivate {
   onSessionProposeResponse(
     topic: string,
     payload: JsonRpcResult<JsonRpcTypes.Results["wc_sessionPropose"]> | JsonRpcError,
+    transportType?: RelayerTypes.MessageEvent["transportType"],
```

Comment:
@ganchoradkov this won't be used, unless we give support in the future. Should i remove it?

Reply:
nevermind, you are using transportTytpe for the subscribe function